### ISSUE TITLE
"Save" only active "onBlur" not "onChange"

### ISSUE THREAD

**Issue by tordans** [bryceco/GoMap#645]
Video: https://share.icloud.com/photos/0e6pOyq6HO4Nu70XLKZpYt0QQ

1. I edit the ref field but the save button stays inactive
2. I tap out of the ref field, now the save button becomes active and I can exit via save
It would be great to be able to save without first switching the input.
I guess this is the same issue of https://github.com/bryceco/GoMap/issues/638, which prevents me from saving the quest.

**Comment by dieterdreist** [comment]
this is also a regression on the all tags tab of an object, where previously you could save after adding a tag, while now you have to hit “done” on the keyboard or trigger onBlur somehow else before you can save.

sent from a phone

**Comment by bryceco** [comment]
Thanks for the report! I noticed this (and other bugs) and have a 3.3.7 in app review now that should fix this. You might want to postpone submitting bug reports until it's released later today. 

I did a bunch of work to take various tag input features that were implemented differently in Common Tags, All Tags and the Quest solver and unified them in a single text field that is now shared by them all. This caused a lot of churn in what is somewhat fragile code but it should make adding new features easier and consistent across those screens.
